### DISCUSSION TITLE
Coverage

### DISCUSSION THREAD

**Discussion by rogerjbos** [block-crawler/block-crawler#29] [General]
What blockchains are covered by block-crawler?  Is it only Ethermeum based chains?  Does it cover Solana, Cosmos, or substrate-based chains like Polkadot?

**Comment by aenglander** [comment]
While the underlying framework pieces, like the data bus and core RPC client, will support any data source, the Block Crawler only has components that interact with EVM-based blockchains today. The long-term plan is to support a multitude of blockchain types. EVM was chosen as the first due to its popularity and the number of blockchains it would enable.
